sizzlingstats/SSTeamData.cpp

```diff
@@ -0,0 +1,79 @@
+
+#include "SSTeamData.h"
+
+void TeamScoreData::Reset()
+{
+	m_AverageMedicDistance.Reset();
+}
+
+CTeamDataManager::CTeamDataManager()
+{
+
+}
+
+CTeamDataManager::~CTeamDataManager()
+{
+}
+
+void CTeamDataManager::Reset()
+{
+	TeamScoreData *tm_data = nullptr;
+	
+	if((tm_data = GetTeamStats(TFTEAM_RED)) != nullptr)
+	{
+		tm_data->Reset();
+	}
+
+	if((tm_data = GetTeamStats(TFTEAM_BLUE)) != nullptr)
+	{
+		tm_data->Reset();
+	}
+
+}
+
+void CTeamDataManager::ResetTeamStatsData()
+{
+}
+
+void CTeamDataManager::AddStatSample( int team, TeamStat stat, float sample_point )
```

Comment:
Maybe also enforce that the team they pass in must be of that enum type. Instead of int team, tfteam team.